From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jelle Raaijmakers <jelle@gmta.nl>
Date: Sat, 29 Apr 2023 15:55:05 +0200
Subject: [PATCH] CMake: Set the install prefix to our staging prefix

This makes sure the binaries are installed to `/usr/local`.
---
 SamTFE/Sources/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/SamTFE/Sources/CMakeLists.txt b/SamTFE/Sources/CMakeLists.txt
index 44bf38675ac1e943bd4fb1532d26b713c8f9c9bd..15922bf2a98fedb843bf83be5d5ddfbdcb7a4fae 100644
--- a/SamTFE/Sources/CMakeLists.txt
+++ b/SamTFE/Sources/CMakeLists.txt
@@ -61,6 +61,7 @@ if(USE_SYSTEM_ZLIB)
 endif()
 
 # RAKE! Where to install the binaries.
+set(CMAKE_INSTALL_PREFIX "${CMAKE_STAGING_PREFIX}")
 if(CMAKE_INSTALL_PREFIX STREQUAL "/usr/local" OR CMAKE_INSTALL_PREFIX STREQUAL "") # Only works for linux since I don't
                                                                                    # know what default is for windows/macos/freebsd.
     set(CMAKE_INSTALL_PREFIX "${CMAKE_SOURCE_DIR}/../") # set install path to project root directory since
